test/fixtures/users.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/Fixtures.html
+
+one:
+  email: MyString
+  password_digest: <%= BCrypt::Password.create('secret') %>
```

Comment:
You are right, though I didn't really look into the test directory as I thought its kind of obsolete now.